simmechanics_to_urdf/firstgen.py

```diff
@@ -1775,7 +1777,7 @@ class URDFGazeboSensorsGenerator:
     def __init__(self):
         self.dummy = ""
 
-    def getURDFForceTorque(self, jointName, sensorName, directionChildToParent, updateRate=100):
+    def getURDFForceTorque(self, jointName, sensorName, directionChildToParent, sensorBlobs, updateRate=100):
```

Comment:
What do you think about removing the hard-coded `updateRate` variable?  
We can either use a global variable and add a line in the readme to say that the update rate is 100 Hz or have it as a parameter in the configuration file (still 100 Hz the default value otherwise we break the compatibility with old models).

Reply:
We agree that the `updateRate` default value of 100Hz in `getURDFForceTorque()` is currently hard-coded in the function signature, but can still be overruled through any function call specifying a new value. Indeed, defining this default value as a "constant" (`UPDATE_RATE = 100`) at the top of the `firstgen.py`script would make it easier to update if necessary.
I fully agree with having it as a parameter in the configuration file and updating the README specifying the default value of 100Hz.
Any objection @traversaro ?

Reply:
I agree completely with @nunoguedelha , but I think this can be done in a new PR.